db/seeds/partials/stocks.rb

```diff
@@ -1,198 +1,10 @@
-    price_cents: 9_325 },
-  { ticker: "UAA",
-    stock_exchange: " ",
-    company_name: "Under Armour",
-    company_website: " ",
-    description: " ",
-    industry: " ",
-    management: " ",
-    employees: 164_000,
-    competitor_names: " ",
-    sales_growth: 2.02,
-    industry_avg_sales_growth: 5.50,
-    debt_to_equity: 1.73,
-    industry_avg_debt_to_equity: 0.85,
-    profit_margin: 23.97,
-    industry_avg_profit_margin: 12.50,
-    cash_flow: 24_000_000_000.00,
-    debt: 95_000_000_000.00,
-    price_cents: 11_223 },
-  { ticker: "GAP",
-    stock_exchange: " ",
-    company_name: "Gap",
-    company_website: " ",
-    description: " ",
-    industry: " ",
-    management: " ",
-    employees: 164_000,
-    competitor_names: " ",
-    sales_growth: 2.02,
-    industry_avg_sales_growth: 5.50,
-    debt_to_equity: 1.73,
-    industry_avg_debt_to_equity: 0.85,
-    profit_margin: 23.97,
-    industry_avg_profit_margin: 12.50,
-    cash_flow: 24_000_000_000.00,
-    debt: 95_000_000_000.00,
-    price_cents: 10_945 },
-  {
-    ticker: "F",
-    stock_exchange: " ",
-    company_name: "Ford",
-    company_website: " ",
-    description: " ",
-    industry: " ",
-    management: " ",
-    employees: 164_000,
-    competitor_names: " ",
-    sales_growth: 2.02,
-    industry_avg_sales_growth: 5.50,
-    debt_to_equity: 1.73,
-    industry_avg_debt_to_equity: 0.85,
-    profit_margin: 23.97,
-    industry_avg_profit_margin: 12.50,
-    cash_flow: 24_000_000_000.00,
-    debt: 95_000_000_000.00,
-    price_cents: 319_000
-  },
-  {
-    ticker: "SONY",
-    stock_exchange: " ",
-    company_name: "Sony",
-    company_website: " ",
-    description: " ",
-    industry: " ",
-    management: " ",
-    employees: 164_000,
-    competitor_names: " ",
-    sales_growth: 2.02,
-    industry_avg_sales_growth: 5.50,
-    debt_to_equity: 1.73,
-    industry_avg_debt_to_equity: 0.85,
-    profit_margin: 23.97,
-    industry_avg_profit_margin: 12.50,
-    cash_flow: 24_000_000_000.00,
-    debt: 95_000_000_000.00,
-    price_cents: 99_231
-  },
-  {
-    ticker: "SIRI",
-    stock_exchange: " ",
-    company_name: "Sirius XM",
-    company_website: " ",
-    description: " ",
-    industry: " ",
-    management: " ",
-    employees: 164_000,
-    competitor_names: " ",
-    sales_growth: 2.02,
-    industry_avg_sales_growth: 5.50,
-    debt_to_equity: 1.73,
-    industry_avg_debt_to_equity: 0.85,
-    profit_margin: 23.97,
-    industry_avg_profit_margin: 12.50,
-    cash_flow: 24_000_000_000.00,
-    debt: 95_000_000_000.00,
-    price_cents: 32_213
-  }
-]
+SEED_TICKERS = %w[AAPL KO SONY DIS SIRI F EA META LUV GAP VZ].freeze
 
-# finding an existing stock or create using only company_name for now
-stocks.each do |stock_data|
-  stock = Stock.find_or_create_by(company_name: stock_data[:company_name])
-  stock.update!(stock_data)
+SEED_TICKERS.each do |ticker|
+  Stock.find_or_create_by(ticker: ticker)
 end
+
+StockPricesUpdateJob.perform_now
```

Comment:
We have a limited number of API calls on the free tier, so we're going to have to leave this as is I think.  Also I think it makes sense to allow people to work on the app and have seed data without having to create an API key.  So for those reasons can you revert the changes in this file please?